tests/load/test_postgresql.py

```diff
@@ -0,0 +1,15 @@
+from load.tasks import BaseDBConnectionBehavior
+
+
+class PostgreSQLConnectionBehavior(BaseDBConnectionBehavior):
+    """
+    This class defines the behavior of a PostgreSQL connection.
+    @TODO: Read query values from sql_queries.json file
+    """
+    db_type = "postgres"
+    table_name = "solar_flare_data"
```

Comment:
Will we be running these tests on a pre-configured database where this table already exists or will we need to add the functionality for creating the table and populating it with some data?

Reply:
Yes, those will be pre populated for now. We will not run any intial seed to load the data. Maybe we can change it later